### ISSUE TITLE
TailwindCSS depends on an outdated version of postcss-load-config

### ISSUE THREAD

**Issue by dykatz** [tailwindlabs/tailwindcss#11081]
TailwindCSS depends on `yaml` indirectly through `postcss-load-config`. [This CVE](https://huntr.dev/bounties/4b494e99-5a3e-40d9-8678-277f3060e96c/) was released earlier today, and was patched in `yaml@2.2.2`. It can be used by `postcss-load-config@4.0.1`, but tailwind depends on `postcss-load-config@^3.1.4`. This is causing our static analysis tools to yell at us.

**Comment by RicardoAALL** [comment]
Yes, same here. Don't know if this is helpful but here is the output from npm audit

```
# npm audit report

yaml  <2.2.2
Severity: moderate
Uncaught Exception in yaml - https://github.com/advisories/GHSA-f9xv-q969-pqx4
fix available via `npm audit fix --force`
Will install tailwindcss@2.2.2, which is a breaking change
node_modules/yaml
  postcss-load-config  3.1.0 - 3.1.4
  Depends on vulnerable versions of yaml
  node_modules/postcss-load-config
    tailwindcss  <=0.0.0-oxide-insiders.ff2c25f || >=2.2.3
    Depends on vulnerable versions of postcss-load-config
    node_modules/tailwindcss

3 moderate severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force 
```

**Comment by zackfencedev** [comment]
Can confirm issue has started today for me as well on all my builds. Attempting to fix downgrades me to version 2.2.2.2.

**Comment by joaoguidev** [comment]
Same here.

**Comment by Arielle-Web** [comment]
same problem for me

**Comment by semy** [comment]
Same here

**Comment by alex-r-redfern** [comment]
Ditto. There is a patch available on the main branch of postcss-load-config if that helps? (see the link in this issue thread: https://github.com/postcss/postcss-load-config/issues/242)

**Comment by thecb4** [comment]
Downgrading didn't help me. And using with SvelteKit to produce a static page is blocked. None of the browsers will load the offending code in a production environment.

**Comment by yp717** [comment]
This is still an issue on our end too!

**Comment by reinink** [comment]
Hey folks!

So the issue here, as suggested, is that we're still using an older version of `postcss-load-config`. The reason for this is because we still supported Node.js v12, meaning we could not upgrade it to a newer version. While technically this CVE is not really relevant to a build-time project like Tailwind, it does cause warnings when using tools like `npm audit`.

Fixing this means updating our minimum Node version requirement, which is something we were hoping to avoid until next major version of the project. However, this issue has forced our hand, and you'll see in #11089 that we've now dropped support for Node.js v12 and have updated any dependencies that were locked to older versions to their latest releases.

Meaning, these changes eliminate the `postcss-load-config` security warning but do make Node.js v14 the new minimum version requirement for Tailwind CSS.

We've just published a new release that includes this fix — `v3.3.2`. Here's how to upgrade:

```sh
npm install tailwindcss@latest
```

We've tested this already using `npm audit`, and it appears to resolve this problem. Please let us know if you bump into any subsequent issues, as we updated a bunch our dependencies as part of this change.
